airflow/bin/cli.py

```diff
@@ -491,16 +491,19 @@ def worker(args):
     from airflow.executors.celery_executor import app as celery_app
     from celery.bin import worker
 
+    concurrency = args.concurrency or conf.get('celery', 'celeryd_concurrency')
```

Comment:
@artwr in this case conf is the ConfigParser, not a dict so it will get the value -- but @bolkedebruin's point is still valid that if args.concurrency is not passed, this line won't properly guard against it. @mistercrunch do you want to change that?